ros1/Dockerfile

```diff
@@ -117,24 +117,30 @@ RUN chmod +x /bin/tini
 
```

Comment:
It looks like there's a few unrelated improvements going on in this file. It might be good to separate those out?